projects/kit/components/input-date/input-date.directive.ts

```diff
@@ -14,11 +14,11 @@ export class TuiInputDateDirective extends AbstractTuiTextfieldHost<TuiInputDate
     }
 
     get max(): TuiDay {
-        return this.host.max;
+        return this.host.max ?? TUI_LAST_DAY;
```

Comment:
We use `computed` getters here.

Reply:
Sorry, not getting what you mean. It is already a getter

Reply:
I mean you can use `this.host.computedMax`, no need to import fallback in that case.